Misc/NEWS.d/next/Security/2024-08-06-11-43-08.gh-issue-80222.wfR4BU.rst

```diff
@@ -0,0 +1,3 @@
+Fix a problem where email.policy.default header refolding could incorrectly
+omit quotes from structured email headers, enabling sender or recipient
+spoofing via a carefully crafted display-name.
```

Comment:
I'd like to be a bit more precise here: "Fix bug in the folding of quoted strings.  Previously when a quoted string was folded so that it spanned more than one line, the surrounding quotes and internal escapes would be omitted.  This could theoretically be used to spoof header lines using a carefully constructed quoted string if the resulting rendered email was transmitted or re-parsed."